### DISCUSSION TITLE
Default support for windows x86

### DISCUSSION THREAD

**Discussion by jbehman-mwb** [actions/runner#2938] [Q&A]
Any reason why Windows x86 builds are not available by default? I was able to generate them by cloning the repo and running the src/dev.cmd with parameters. I tested the runner on x86 Windows machine and all worked fine.

**Comment by Sharmapankaj3187** [comment]
Hi @jbehman-mwb did you run your windows runners in k8s cluster / ARC (Action Runner Controller) setup? https://github.com/actions/actions-runner-controller

**Comment by jbehman-mwb** [comment]
I stood up the environment in aws.
